client/utils/augmentUtils.ts

```diff
@@ -203,3 +179,80 @@ const evaluateCondition = (
       return false;
   }
 };
+
+export const applyAugmentEffects = (
+  baseFantasyPoints: number,
+  stats: GameStats,
+  playerId: string,
+  qualifyingPlayers: Player[] | undefined,
+  augment: Augment | undefined,
+): number => {
+  if (!augment || !augment.isActive || !qualifyingPlayers) {
+    return baseFantasyPoints;
+  }
+
+  const playerQualifies = qualifyingPlayers.some((p) => p.id === playerId);
+  if (!playerQualifies) {
+    return baseFantasyPoints;
+  }
+
+  if (!augment.effects || augment.effects.length === 0) {
+    return baseFantasyPoints;
+  }
+
+  let totalPoints = baseFantasyPoints;
+
+  augment.effects.forEach((effect) => {
+    effect.statBoosts.forEach((boost) => {
+      const statValue = getStatValue(stats, boost.stat);
+      const baseStatMultiplier = getBaseStatMultiplier(boost.stat);
```

Comment:
Do you think it makes sense to combine these two functions into something like `getBaseStatFantasyPoints` or something? The name could use some work lol, but both use the same switch statement so we could use the same code paths

Reply:
Structurally they are similar but I feel like it's best to keep them separate since they serve different functions. For example, `getStatValue` requires different arguments, returns a number similar to the other function, but its only used to get the player stat. The other returns the multiplier, and there are cases where you may need one and not the other.

However, since `getBaseStatMultiplier` is basically a function acting like a map, I created a static record object instead where we just refer to that using the key like `points` or `rebounds`.

```
const STAT_MULTIPLIERS: Record<string, number> = {
  points: 1,
  rebounds: 1,
  assists: 2,
  steals: 3,
  blocks: 3,
  turnovers: -2,
};
```
and the implementation is essentially

```suggestion
      const statValue = getStatValue(stats, boost.stat);
      const baseStatMultiplier = STAT_MULTIPLIERS[boost.stat] ?? 0;
```

Reply:
I see, that's good then. Another side comment, do you think it's better if we calculate the fantasy points in here rather than pass it in then? We already have the whole `GameStats` object